test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -171,20 +168,23 @@ bool UcxContext::init()
     worker_params.field_mask  = UCP_WORKER_PARAM_FIELD_THREAD_MODE;
     worker_params.thread_mode = UCS_THREAD_MODE_SINGLE;
     status = ucp_worker_create(_context, &worker_params, &_worker);
-    if (status == UCS_OK) {
-        status = epoll_init();
-        if (status != UCS_OK) {
-            destroy_worker();
-            _worker = NULL;
-        }
-    }
     if (status != UCS_OK) {
         ucp_cleanup(_context);
         _context = NULL;
         UCX_LOG << "ucp_worker_create() failed: " << ucs_status_string(status);
         return false;
     }
 
+    status = epoll_init();
+    if (status != UCS_OK) {
+        destroy_worker();
+        _worker = NULL;
+        ucp_cleanup(_context);
+        _context = NULL;
+        return false;
+
```

Comment:
remove space line

Reply:
sorry for adding the obvious extra space line here.